prody/proteins/interactions.py

```diff
@@ -712,7 +715,11 @@ def calcInteractionMatrix(atoms):
     resIDs = list(atoms.select('name CA').getResnums())
     resChIDs = list(atoms.select('name CA').getChids())
     resIDs_with_resChIDs = list(zip(resIDs, resChIDs))
-    scoring = [2, 3, -1, 3, 3, 1]  #HBs_calculations, SBs_calculations, SameChargeResidues, Pi_stacking, Pi_cation, Hydroph_calculations
+
+    try:
+        scoring = kwargs['scoring']
+    except:
+        scoring = [2, 3, -1, 3, 3, 1]  #HBs_calculations, SBs_calculations, SameChargeResidues, Pi_stacking, Pi_cation, Hydroph_calculations
```

Comment:
You can set the default values from kwargs.get

also, it might be better to have separate kwargs for each so people don’t have to remember the order and defaults

Reply:
The code would look as follows:
```
scoring = kwargs.get('scoring', [2, 3, -1, 3, 3, 1])  #HBs_calculations, SBs_calculations, SameChargeResidues, Pi_stacking, Pi_cation, Hydroph_calculations
```